### ISSUE TITLE
Overview link broken in 2-algo html

### ISSUE THREAD

**Issue by brockho** [numbbo/coco#1355]
The overview link in the "Runtime distributions (ECDFs) summary and function groups" page of the html output when comparing two algorithms is broken: instead of linking to to `templateBBOBcmp.html` it links to `templateBBOBmany.html`. Maybe, we should refrain from using hard links (hard to maintain anyway) and instead opt for a simple `back` button, e.g. via:
```
<FORM><INPUT Type="button" VALUE="Back" onClick="history.go(-1);return true;"></FORM>
```
[to be tested of course]

**Comment by dtusar** [comment]
The ECDF pages for two and more algorithms are generated in the same folder. Hence, they are overwritten after each new run. I would suggest that we avoid the confusion by just writing the files in different folders.

**Comment by dtusar** [comment]
Solved with #1396.
